perception/image_projection_based_fusion/src/roi_cluster_fusion/node.cpp

```diff
@@ -41,8 +41,10 @@ RoiClusterFusionNode::RoiClusterFusionNode(const rclcpp::NodeOptions & options)
   use_iou_ = declare_parameter("use_iou", false);
   use_cluster_semantic_type_ = declare_parameter("use_cluster_semantic_type", false);
   iou_threshold_ = declare_parameter("iou_threshold", 0.1);
+  unknown_iou_threshold_ = declare_parameter("unknown_iou_threshold", 0.1);
```

Comment:
[imo] Could you add descriptions about these parameters to document?

Reply:
I added description here https://github.com/autowarefoundation/autoware.universe/pull/4123/commits/882859143d6b1ab6e88a8f6f100a394cd4738b21